examples/dataflow-hashpipeline/test/generate-input-file.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google Inc.
```

Comment:
Please move these files to the `scripts/` dir.
Reasoning: 
There are helpful for demo-purposes.